Libraries/Text/RCTShadowText.m

```diff
@@ -105,6 +111,8 @@ - (NSAttributedString *)_attributedStringWithFontFamily:(NSString *)fontFamily
   [self _addAttribute:RCTReactTagAttributeName withValue:self.reactTag toAttributedString:attributedString];
   [self _setParagraphStyleOnAttributedString:attributedString];
 
+  [self _addAttribute:NSKernAttributeName withValue:@(letterSpacing) toAttributedString:attributedString];
```

Comment:
This is a tiny nit, but I would move this to after the line that adds the `NSFontAttributeName` attribute to maintain consistency but that's really the only thing I have against this. Looks good otherwise! :+1: